security-analysis/src/main/java/com/powsybl/security/Security.java

```diff
@@ -301,6 +509,18 @@ private static float getViolationValue(LimitViolation violation) {
         return Math.abs(violation.getValue()) / violation.getLimit() * 100f;
     }
 
+    private static NumberFormat getFormatter(Locale locale, int minimumFractionDigits, int maximumFractionDigits) {
+        NumberFormat numberFormat = NumberFormat.getNumberInstance(locale);
+        numberFormat.setMinimumFractionDigits(minimumFractionDigits);
+        numberFormat.setMaximumFractionDigits(maximumFractionDigits);
+        numberFormat.setGroupingUsed(false);
+        return numberFormat;
+    }
+
+    /**
+     * @deprecated Use getFormatter(Locale, int, int) instead.
+     */
+    @Deprecated
```

Comment:
Remove this method: it's private

Reply:
I can't. The method is still use by a @Deprecated public method. I'll do it after I'll clean up the code in the ipts.